spec/controllers/rooms_controller_spec.rb

```diff
@@ -59,9 +59,29 @@
       expect(JSON.parse(response.body)['data']['join_url']).to eq(join_url)
     end
 
-    it 'returns not_found if the room doesn\'t exist' do
+    it 'returns :not_found if the room doesn\'t exist' do
       post :start, params: { friendly_id: 'invalid_friendly_id' }
       expect(response).to have_http_status(:not_found)
     end
   end
+
+  describe '#create' do
+    let(:room_params) do
+      {
+        room: { name: Faker::Science.science }
+      }
+    end
+
+    it 'creates a room for the authenticated user' do
+      session[:user_id] = user.id
+      expect { post :create, params: room_params }.to change { user.reload.rooms.count }.from(0).to(1)
```

Comment:
Is the `reload` necessary here?

Reply:
After checking, It appears that it is not required.
The `count` method seems to be making its own `SQL query`.